distribution/lib/Standard/Database/0.0.0-dev/src/Internal/Database_Type_Helpers.enso

```diff
@@ -0,0 +1,11 @@
+from Standard.Base import all
+
+from Standard.Table.Data.Type.Value_Type_Helpers import Operation_Type_Helpers
+
+import project.Data.Column.Column
+
+## PRIVATE
+   Type helpers for database tables.
+type_helpers =
+    is_column value = value.is_a Column
+    Operation_Type_Helpers.Instance is_column
```

Comment:
No idea what this is for could we have a little more detail on the doc string please.